javascript-data-model/order-history.js

```diff
@@ -0,0 +1,35 @@
+var orderHistory = [
+  {
+    item: 'Javascript for impatient programmers',
+    orderDate: 'Aug 4, 2020',
+    orderNumber: '114-3941689-8772232',
+    price: 34,
+    returnWindow: 'Sep 7, 2020',
+    deliverDate: 'Aug 8, 2020'
+  },
+  {
+    item: 'The Timeless Way of Building',
+    orderDate: 'July 19, 2020',
+    orderNumber: '113-9984268-1280257',
+    price: 44.53,
+    returnWindow: 'Aug 19, 2020',
+    deliverDate: 'Jul 20, 2020'
+  },
+  {
+    item: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter',
+    orderDate: 'July 4, 2020',
+    orderNumber: '114-2875557-9059409',
+    price: 17.22,
+    returnWindow: 'Aug 5, 2020',
+    deliverDate: 'July 7, 2020'
+  },
+  {
+    item: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+    itemAdditional: 'The Art of Sql',
+    orderDate: 'July 3, 2020',
+    orderNumber: '113-2883177-2648248',
+    price: 138.93,
+    returnWindow: 'Aug 4, 2020',
+    deliverDate: 'July 5, 2020'
+  }
+];
```

Comment:
Few things to fix here:
1. We are missing price, image, returnWindow per item, not per order
2. More than one item need to be somehow connect to one order. For instance, the last order has two items